Updated terminologies for chaos agent, workflow and charts (#197)
litmuschaos/litmus-docs: website/docs/getting-started/run-your-first-workflow.md

---
id: run-your-first-workflow
title: Run your First Chaos Scenario in 5 minutes

> sidebar_label: Run Your FirstChaos Scenario

Reviewer: ```suggestion
sidebar_label: Run Your First Chaos Scenario
```

---

## Schedule your FirstChaos Scenario

> ### Select the podtato-head predefinedChaos Scenario

Reviewer: ```suggestion
### Select the podtato-head predefined Chaos Scenario
```

> 1. Click on <span style={{color: '#5B44BA'}}>**Schedule aChaos Scenario**</span> from the ChaosCenter Homepage or from the top right button in the **Litmus Workflows** tab.

Reviewer: ```suggestion
1. Click on <span style={{color: '#5B44BA'}}>**Schedule a Chaos Scenario**</span> from the ChaosCenter Homepage or from the top right button in the **Litmus Workflows** tab.
```

<br /> <br />

2. Select **SelfChaos Delegate** as the target [Chaos Delegate](resources.md#chaosagents) for Chaos Injection. This is where we'll select which Chaos Delegate to choose as the Chaos Delegate.

   <img src={require("../assets/getting-started/run-your-first-workflow/selecting-self-agent.png").default} alt="Selecting Self Chaos Delegate" />

3. Expand the first radio button (To create a new Chaos Scenario from Predefined Chaos Scenario Templates) and select **podtato-head** from the list of Predefined Chaos Scenarios.

   <img src={require("../assets/getting-started/run-your-first-workflow/select-podtato-head.png").default} alt="Select Podtato Head" />

4. View the Chaos Scenario details in the **Chaos Scenario Settings**, you can modify the name and description of the Chaos Scenario to suit your needs.

   <img src={require("../assets/getting-started/run-your-first-workflow/workflow-settings-podtato-head.png").default} alt="Modify Details of Podtato Head Chaos Scenario" />

> ### Simulate theChaos Scenario Steps Visualization

Reviewer: ```suggestion
### Simulate the Chaos Scenario Steps Visualization
```

5. View the visualization of the Litmus Chaos Scenario you are about to execute. This step also allows for you to edit or modify the YAML/tunable if required. We would just stick with the default configurations for now.

   <img src={require("../assets/getting-started/run-your-first-workflow/podtato-head-tune-workflow.png").default} alt="Visualize Podtato Head Chaos Scenario" />

   > By default in the Podtato Head Chaos Scenario Template the steps to gracefully delete the Chaos Resources (`revert-chaos`) and also the Podtato Head application (`delete-application`) are present.

> ### Assign Weights to theChaos Scenario Experiments

Reviewer: ```suggestion
### Assign Weights to the Chaos Scenario Experiments
```

6. Assign weights to the chaos experiments that are part of theChaos Scenario using the slider. This is typically used when there are multiple experiments as part of a Chaos Scenario. These weights influence the **Resilience Score** calculation for the Chaos Scenario.

   <img src={require("../assets/getting-started/run-your-first-workflow/adjust-weights-for-podtato-head.png").default} alt="Assign Weights for Chaos Scenario Experiments" />

> ### Schedule the Podtato-HeadChaos Scenario

Reviewer: ```suggestion
### Schedule the Podtato-Head Chaos Scenario
```

7. Schedule the Litmus Chaos Scenario for immediate and one-time execution by selecting the **Schedule Now** option

   <img src={require("../assets/getting-started/run-your-first-workflow/choose-schedule.png").default} alt="Choose a Schedule for Chaos Scenario Execution" />

8. Verify and click on **Finish** to start the Chaos Injection

>    <img src={require("../assets/getting-started/run-your-first-workflow/verify-and-commit-podtato-head.png").default} alt="Verify and Schedule the Podtato HeadChaos Scenario" />

Reviewer: ```suggestion
   <img src={require("../assets/getting-started/run-your-first-workflow/verify-and-commit-podtato-head.png").default} alt="Verify and Schedule the Podtato Head Chaos Scenario" />
```

### Congratulations

And with that you have successfully scheduled your first Chaos Scenario with Litmus.

<img src={require("../assets/getting-started/run-your-first-workflow/scheduled-successfully.png").default} alt="Podtato Head Chaos Scenario Scheduled Successfully" />

---

## Visualize and Analyze

> 1. To check the current progress of the Podtato-HeadChaos Scenario, view the status of theChaos Scenario from the **Litmus Chaos Scenarios** Tab.

Reviewer: ```suggestion
1. To check the current progress of the Podtato-Head Chaos Scenario, view the status of the Chaos Scenario from the **Litmus Chaos Scenarios** Tab.
```

   podtato-head-chaos-1622018315-80657052      0/2     Completed           0          79s
   podtato-head-pod-delete-chaosx55vh-runner   1/1     Running             0          25s
   subscriber-958948965-qbx29                  1/1     Running             0          6h16m

>    Chaos Scenario-controller-78fc7b6c6-w82m7         1/1     Running             0          6h16m

Reviewer 2: This can be in small letters, representing a pod-name

>    Chaos Scenario-controller-78fc7b6c6-w82m7         1/1     Running             0          6h16m

Reviewer: We can revert this change
